components/Common/MetaBar/index.module.css

```diff
@@ -0,0 +1,65 @@
+.wrapper {
+  @apply flex
+    w-80
+    flex-col
+    items-start
+    gap-8
+    border-l
+    border-neutral-200
+    p-6
+    dark:border-neutral-900;
+
+  dt {
+    @apply mb-2
+      text-sm
+      font-medium
+      text-neutral-800
+      dark:text-neutral-200;
+  }
+
+  dd {
+    @apply mb-8
+      flex
+      items-center
+      gap-2
+      text-sm
+      text-neutral-900
+      dark:text-white;
+
+    a {
+      @apply font-semibold
+        !text-neutral-900
```

Comment:
Not a fan of important statements, is there any way to not use something like that in this case? 🙈

Reply:
I will check the import order of the styles 👍